controllers/account/ec2.go

```diff
@@ -43,7 +39,7 @@ func (r *AccountReconciler) InitializeSupportedRegions(reqLogger logr.Logger, ac
 	// We should not bomb out just because we can't retrieve the vCPU value
 	// and we'll just continue with a "0"
 	// Errors are logged already in getDesiredVCPUValue
-	vCPUQuota, _ := r.getDesiredVCPUValue(reqLogger)
+	vCPUQuota, _ := r.getDesiredServiceQuotaValue(reqLogger, "vcpu")
```

Comment:
We should validate which types we want to use where, either string or the new servicequota types